File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/multiline/uncomment/direct/Java/Go/s380356166_commented.go
package main

import (
	"fmt"
)

func main() {
	var K, S int
	fmt.Scanf("%d %d", &K, &S)
	var ans int
	for x := 0; x <= K; x++ {
		for y := 0; y <= K; y++ {
			z := S - x - y
			if z >= 0 && z <= K {
				ans++
			}
		}
	}
	fmt.Println(ans)
}
// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/direct/comment/uncomment/direct/Java/C/s380356166_commented.c
#include <stdio.h>

int main() {
    int K, S;
    scanf("%d %d", &K, &S);
    
    int ans = 0;
    for (int x = 0; x <= K; x++) {
        for (int y = 0; y <= K; y++) {
            int z = S - x - y;
            if (z >= 0 && z <= K) {
                ans++;
            }
        }
    }
    
    printf("%d\n", ans);
    return 0;
}
// End of Code
